docs/editors/overview.md

```diff
@@ -38,181 +39,207 @@ functionality.
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Goto definition</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Goto type definition</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Completions</td>
     <td align="center">✅</td>
     <td align="center">✅*</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Hover</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Hover for selection</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">❌</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Parameter hints</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Find references</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Run/Debug</td>
     <td align="center">✅</td>
     <td align="center"></td>
     <td align="center"></td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Find implementations</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Rename symbol</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Code actions</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">✅</td>
   </tr>
   <tr>
     <td>Worksheets</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
   </tr>
   <tr>
     <td>Document symbols</td>
     <td align="center">✅</td>
     <td align="center">Flat</td>
     <td align="center">Flat</td>
     <td align="center">✅</td>
+    <td align="center">Flat</td>
   </tr>
   <tr>
     <td>Workspace symbols</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
     <td align="center">✅</td>
+    <td align="center">❌</td>
```

Comment:
Wait, is that right, Helix doesn't support workspace symbol search? I'm pretty sure that there's a picker for that and it seemed to work with Metals last time I checked. :slightly_smiling_face:

Reply:
I tested it out on a small project and it didn't seem to work for me (dialogue was empty) but if you're able to re-test and get it working then I'm happy to update this

Reply:
I just tested it again right now and looks like it works fine (SBT project with Bloop BSP). Don't know if there are some hidden limitations here. :slightly_smiling_face:

Reply:
One thing that it might have been is that by default some clients try to request everything from the server and Metals won't respond to that because you'd just be returning soooo many symbols. So it requires a query.